Index: src/drivers/net/wireless/intel/iwlwifi/iwl-drv.c
===================================================================
--- src.orig/drivers/net/wireless/intel/iwlwifi/iwl-drv.c	2019-09-06 19:04:21.667401871 +0200
+++ src/drivers/net/wireless/intel/iwlwifi/iwl-drv.c	2019-09-06 19:24:21.498781984 +0200
@@ -89,6 +89,7 @@
 MODULE_DESCRIPTION(DRV_DESCRIPTION);
 MODULE_AUTHOR(DRV_COPYRIGHT " " DRV_AUTHOR);
 MODULE_LICENSE("GPL");
+MODULE_VERSION("4.18.0-107_dup8.0");
 
 #ifdef CONFIG_IWLWIFI_DEBUGFS
 static struct dentry *iwl_dbgfs_root;
Index: src/drivers/net/wireless/intel/iwlwifi/dvm/main.c
===================================================================
--- src.orig/drivers/net/wireless/intel/iwlwifi/dvm/main.c	2019-06-27 14:54:04.115678555 +0200
+++ src/drivers/net/wireless/intel/iwlwifi/dvm/main.c	2019-09-06 19:25:01.896353376 +0200
@@ -55,6 +55,7 @@
 MODULE_DESCRIPTION(DRV_DESCRIPTION);
 MODULE_AUTHOR(DRV_COPYRIGHT " " DRV_AUTHOR);
 MODULE_LICENSE("GPL");
+MODULE_VERSION("4.18.0-107_dup8.0");
 
 /* Please keep this array *SORTED* by hex value.
  * Access is done through binary search.
Index: src/drivers/net/wireless/intel/iwlwifi/mvm/ops.c
===================================================================
--- src.orig/drivers/net/wireless/intel/iwlwifi/mvm/ops.c	2019-09-06 19:04:21.306410478 +0200
+++ src/drivers/net/wireless/intel/iwlwifi/mvm/ops.c	2019-09-06 19:25:25.922098470 +0200
@@ -88,6 +88,7 @@
 MODULE_DESCRIPTION(DRV_DESCRIPTION);
 MODULE_AUTHOR(DRV_COPYRIGHT " " DRV_AUTHOR);
 MODULE_LICENSE("GPL");
+MODULE_VERSION("4.18.0-107_dup8.0");
 
 static const struct iwl_op_mode_ops iwl_mvm_ops;
 static const struct iwl_op_mode_ops iwl_mvm_ops_mq;
